test/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+      "no-unused-expressions": [0] // because chai's expect is weird
```

Comment:
This is because a lot of chai expectations, like `expect(...).to.be.equal;` use getter methods instead of functions :\